.github/workflows/ci.yml

```diff
@@ -148,7 +148,8 @@ jobs:
 
   run-integration-tests:
     name: Run Integration Tests (${{ matrix.component.Name }})
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
+    container: ghcr.io/eclipse-velocitas/devcontainer-base-images/python:latest
```

Comment:
Should we use latest here or better reference a defined version?

Reply:
Right now I would like to go with latest since it *should* be the same version that is also used for the devContainer which is used to develop the application. If we fix the version for the devContainer, we should also fix it here, I agree.